src/demo/pages/PickersPage/examples/PeoplePicker.Types.Example.tsx

```diff
@@ -25,46 +22,11 @@ const suggestionProps: IBasePickerSuggestionsProps = {
 
 export class PeoplePickerTypesExample extends React.Component<any, IPeoplePickerExampleState> {
   private _peopleList;
-  private contextualMenuItems: IContextualMenuItem[] = [
-    {
-      key: 'newItem',
-      icon: 'circlePlus',
-      name: 'New'
-    },
-    {
-      key: 'upload',
-      icon: 'upload',
-      name: 'Upload'
-    },
-    {
-      key: 'divider_1',
-      name: '-',
-    },
-    {
-      key: 'rename',
-      name: 'Rename'
-    },
-    {
-      key: 'properties',
-      name: 'Properties'
-    },
-    {
-      key: 'disabled',
-      name: 'Disabled item',
-      isDisabled: true
-    }
-  ];
 
   constructor() {
     super();
     this._onFilterChanged = this._onFilterChanged.bind(this);
-    this._peopleList = [];
-    people.forEach((persona: IPersonaProps) => {
-      let target: IPersonaWithMenu = {};
-
-      assign(target, persona, { menuItems: this.contextualMenuItems });
-      this._peopleList.push(target);
-    });
+    this._peopleList = people;
```

Comment:
seems confusing for an example. Is people a global?

The global could be underscored, it's private. I was looking for a "people" local variable because it doens't have an underscore.